lib/fastlane/actions/xcodebuild.rb

```diff
@@ -112,8 +120,13 @@ def self.run(params)
           if hash_args = hash_to_args(params)
             xcodebuild_args += hash_args
           end
+
+          buildlog_path ||= params[:buildlog_path]
         end
 
+        # By default we put xcodebuild.log in the current folder
+        buildlog_path ||= "."
```

Comment:
Have we been doing that up until now? How about we store it in `~/Library/Logs/xctest` by default?